### ISSUE TITLE
Document all `py2exe` settings in `setup.py`

### ISSUE THREAD

**Issue by abitrolly** [py2exe/py2exe#53]
Wiki is nice (although there is also one that should be disabled in GitHub settings). There is still a need for a proper documentation that can be synced with `py2exe` releases.

For example, in [the tutorial](http://www.py2exe.org/index.cgi/Tutorial#Step2) I could not find a link to the all `py2exe` specific options added to `setup()`. What I needed, however, is this block.

https://github.com/py2exe/py2exe/blob/b531de22b7faa45eee29266a3c5a852fd7c92920/py2exe/setup_template.py#L145-L170

It took a while to figure out that `console=` is `py2exe` specific.
```python
from distutils.core import setup
import py2exe

setup(console=['hello.py'])
```

**Comment by albertosottile** [comment]
Thanks for reporting this and thanks for the hint, I disabled the wiki on GitHub.

Regarding your suggestion, I would like very much to have an up to date documentation of `py2exe`. As you noticed, the website content is almost all outdated and it would benefit from a serious review. Unfortunately, I do not have the time to do so properly. 

If you are willing to help, I would really appreciate any contribution in this regard, e.g. an updated tutorial or a new list and description of the supported keywords of `setup`. Some of this information is scattered in the codebase, as you found out. For the missing parts, I could help. This work would also assist me in tracking all the features available and prioritize what to support and what to drop in the future.

For the moment, I am pinning this issue and leaving it open with the Help Wanted label. Let's hope that someone will pick it up.

**Comment by pintergreg** [comment]
I personally would use the GH Wiki or rather Pages for a simple static site instead of the current one.
I think it is easier to contribute via PR-s even if its documentation.

**Comment by abitrolly** [comment]
+1 for statically generated web site.

**Comment by albertosottile** [comment]
@pintergreg and for everyone else interested in contributing to this: I just enabled the GH Wiki on this repository.

**Comment by albertosottile** [comment]
To some extent this is fixed in 0.12.0.0: a new `freeze` API was introduced, which replaces the former `distutils`-based API, and its documentation is available here: https://github.com/py2exe/py2exe/blob/master/docs/py2exe.freeze.md . Hence, for now I am closing this issue. if you have any suggestions on how to improve the new documentation, I would encourage you to open a new one.
